src/main/java/appeng/capabilities/NullSpatialDimension.java

```diff
@@ -0,0 +1,54 @@
+
+package appeng.capabilities;
+
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3i;
+import net.minecraft.world.World;
+
+import appeng.api.storage.ISpatialDimension;
+
+
+class NullSpatialDimension implements ISpatialDimension
+{
+	@Override
+	public int createNewCellDimension( Vec3i size, int owner )
+	{
+		return -1;
+	}
+
+	@Override
+	public void deleteCellDimension( int cellStorageId )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)